CONTRIBUTING.md

```diff
@@ -18,13 +18,36 @@ Welcome! We're so excited that you're considering contributing to the official G
 ---
 
 ## Commit your update
+Please follow our project commit guide lines below:
+- The first line is the title of the message,
+it should be no longer than 63 characters.
+- The second line should be left empty.
+- The remaining consist of the body of the message,
+they should be no longer than 72 characters each.
+try to be descriptive as you can while keeping it simple.
+
+- Example:
+Adding an awesome feature.
+
+Adding feature which:
+1. makes cookies.
+2. makes you get A+ in all courses.
+3. create dank memes.
+in order to use each of the feature above just go to page ****,
+write the line "make_cookies()" where ever you want and walla, 
+cookies will appear.
+
 Commit the changes once you are happy with them. 
 
 ---
+## Review requirements 
+- Before revewing the task, please acknowledge its requirements and acceptance criteria.
+
 
+--- 
 ## Pull Request
 - When you're finished with the changes, create a pull request, also known as a PR.
-- Please note that at least two team members of the group should review and approve the pull request before asking the mentors to review and merge.
+- Please note that at least two team members of the group should quality review(NO LGTM) and approve the pull request before asking the mentors to review and merge.
```

Comment:
NO LGTM ,should be betwen us but not in public i think you should delete it

Reply:
Nothing should be kept private between you. You should consider this project as any open source project; if someone new would like to join the effort - this doc should let him know everything about how to contribute. CR process included.